newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
+
+int main1()
+{
+	int arr[N], number, i, flag = 1, counter = 0;
+
+	while (flag)
+	{
+		printf("Enter %d integer numbers, please: ", N);
+
+		for (i = 0; i < N && flag; i++)
+			if (scanf("%d", &arr[i]) != 1)
+				return 1;
```

Comment:
Usually in case of error, the return value is negative (-1 for example).
Also, define a const with this value and return the const, it is more readable.